generators/index.js

```diff
@@ -154,7 +154,7 @@ module.exports = class extends Generator {
         packageName: packageName,
         gulpFactoryLocation: gulpFactoryLocation,
         rollupConfigLocation: rollupConfigLocation,
-        testFileLocation: testFileLocation
+        testFileLocation: testFileLocation,
```

Comment:
Why the addition of the trailing comma?  I think this could cause issues if you're on older versions of node

Reply:
Typo. Just removed it. Thanks, Cassondra! :)